force_wfmanager/left_side_pane/view_utils.py

```diff
@@ -44,3 +45,13 @@ def _item_info_from_group(group_contents, item_info=None):
         elif isinstance(entity, Item):
             item_info.append(entity.name)
     return item_info
+
+
+def get_default_background_color():
+    """Return the default background color for a Qt Application"""
+    palette = QtGui.QPalette()
+    rgba_color = palette.window().color().getRgb()
```

Comment:
This might work, but I would have used the QApplication.palette().getColor(Active, Background)

They _might_ be equivalent, because the new palette uses the application palette by default as a prototype, but I am not 100% sure about the window() call.

my Qt is rusty though.

Reply:
Question: what happens if you don't specify a background color? will it default to white? We might not need this.